testing/test_runner.py

```diff
@@ -11,11 +14,17 @@
 from _pytest import outcomes
 from _pytest import reports
 from _pytest import runner
+from _pytest.outcomes import Exit
+from _pytest.outcomes import Failed
 from _pytest.outcomes import OutcomeException
+from _pytest.outcomes import Skipped
+
+if False:  # TYPE_CHECKING
+    from typing import Type
 
 
 class TestSetupState:
-    def test_setup(self, testdir):
+    def test_setup(self, testdir) -> None:
```

Comment:
Wouldn't it be good to also add a type annotation for fixtures, i.e. "testdir" here?
In the long run it would be good to have this via a mypy plugin automatically (https://github.com/python/mypy/issues/6760).

Reply:
Definitely makes sense. In this PR in this file I just did a mechanical `-> None` + fixups. Fixtures like `testdir` can have a mechanical treatment too but maybe in another PR. I'm hoping to get this one merged so I can continue with more typing work.